src/js/view/view.js

```diff
@@ -752,6 +768,9 @@ function View(_api, _model) {
 
         addClass(_playerElement, 'jw-flag-ads');
         removeClass(_playerElement, 'jw-flag-live');
+        if (_controls) {
```

Comment:
There's already a check to see if controls exist in line 776. You can call `addGradient` there before `_controls.userActive();`.